tests/test_glacier.py

```diff
@@ -266,3 +266,57 @@ def test_glacier_vault_delete(self):
         with self.assertRaises(ClientError) as e:
             client.describe_vault(vaultName='c7n-test-delete')
         self.assertEqual(e.exception.response['Error']['Code'], 'ResourceNotFoundException')
+
+
+class TestCEL(BaseTest):
+    @functional
+    def test_glacier_cel_cross_account(self):
+        session_factory = self.replay_flight_data("test_glacier_remove_matched")
+        client = session_factory().client("glacier")
+        name = "test-glacier-cel-cross-account"
+        client.create_vault(vaultName=name)
```

Comment:
Does this cleanup and creation of a vault need to happen in unit test? See if this can be simplfied